napari/layers/_image_layer/model.py

```diff
@@ -98,7 +97,7 @@ class Image(Layer):
     _colormaps = AVAILABLE_COLORMAPS
 
     default_colormap = 'magma'
-    default_interpolation = 'nearest'
+    default_interpolation = Interpolation.NEAREST
```

Comment:
I would consider this "public" so this should be the string version:
```suggestion
    default_interpolation = str(Interpolation.NEAREST)
```

Reply:
Fixed. Thanks!